### PR TITLE
Non ugent vo deployment hpc 2178

### PR DESCRIPTION
**Pull request by itkovian**
- create VO filesets for gvo000[16,17,18]
- create member directories for these VOs
- set quota on both the filesets and the member directories

### CODE REVIEW

**Comment by itkovian** [comment]
Right now, we deploy on _all_ scratch. Maybe this is not the desired behaviour.

**Comment by stdweird** [comment]
it's ok for me if you do this on all scratch
is this tested somewhere?

**Comment by itkovian** [comment]
Not yet, since the GPFS issue needs t be addressed first.

**Comment by itkovian** [comment]
Requires https://github.com/hpcugent/vsc-filesystems/pull/41

**Comment by itkovian** [comment]
Tested on storage10 for gvo00016 (brussels users):

~~~~
(vsc)[root@storage10 ~]# ls -l /user/scratchdelcatty/gent/gvo000/gvo00016/
total 0
drwx------ 2 vsc10002 vsc10002 4096 Nov 27 12:03 vsc10002
drwx------ 2 vsc10003 vsc10003 4096 Nov 27 12:03 vsc10003
drwx------ 2 vsc10004 vsc10004 4096 Nov 27 12:03 vsc10004
drwx------ 2 vsc10005 vsc10005 4096 Nov 27 12:03 vsc10005
drwx------ 2 vsc10006 vsc10006 4096 Nov 27 12:03 vsc10006
drwx------ 2 vsc10007 vsc10007 4096 Nov 27 12:03 vsc10007
drwx------ 2 vsc10008 vsc10008 4096 Nov 27 12:03 vsc10008
drwx------ 2 vsc10009 vsc10009 4096 Nov 27 12:03 vsc10009
drwx------ 2 vsc10010 vsc10010 4096 Nov 27 12:03 vsc10010
drwx------ 2 vsc10011 vsc10011 4096 Nov 27 12:03 vsc10011
drwx------ 2 vsc10012 vsc10012 4096 Nov 27 12:03 vsc10012
drwx------ 2 vsc10013 vsc10013 4096 Nov 27 12:03 vsc10013
drwx------ 2 vsc10014 vsc10014 4096 Nov 27 12:03 vsc10014
drwx------ 2 vsc10015 vsc10015 4096 Nov 27 12:03 vsc10015
drwx------ 2 vsc10016 vsc10016 4096 Nov 27 12:03 vsc10016
drwx------ 2 vsc10017 vsc10017 4096 Nov 27 12:03 vsc10017
drwx------ 2 vsc10018 vsc10018 4096 Nov 27 12:03 vsc10018
drwx------ 2 vsc10019 vsc10019 4096 Nov 27 12:03 vsc10019
drwx------ 2 vsc10020 vsc10020 4096 Nov 27 12:04 vsc10020
drwx------ 2 vsc10021 vsc10021 4096 Nov 27 12:04 vsc10021
drwx------ 2 vsc10022 vsc10022 4096 Nov 27 12:04 vsc10022
drwx------ 2 vsc10023 vsc10023 4096 Nov 27 12:04 vsc10023
drwx------ 2 vsc10024 vsc10024 4096 Nov 27 12:04 vsc10024
drwx------ 2 vsc10025 vsc10025 4096 Nov 27 12:04 vsc10025
drwx------ 2 vsc10026 vsc10026 4096 Nov 27 12:04 vsc10026
~~~~

File: test/vo.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+        Options = namedtuple("Options", ['dry_run'])
+        options = Options(dry_run=False)
+
+        mock_client.return_value = mock.MagicMock()
+        storage = None  # not used anymore, it seems
+
+        for storage_name in (VSC_HOME, VSC_DATA, VSC_SCRATCH_DELCATTY, VSC_SCRATCH_PHANPY):
+            with mock.patch('vsc.administration.vo.VscTier2AccountpageVo', autospec=True) as mock_vo, \
+                 mock.patch('vsc.administration.vo.VscTier2AccountpageUser', autospec=True) as mock_user, \
+                 mock.patch('vsc.administration.vo.update_vo_status') as mock_update_vo_status:
```
Comment (boegel) [inline test/vo.py]: to make this py2.6 compatible:

```python
with mock.patch('vsc.administration.vo.VscTier2AccountpageVo', autospec=True) as mock_vo:
    with mock.patch('vsc.administration.vo.VscTier2AccountpageUser', autospec=True) as mock_user:
        with mock.patch('vsc.administration.vo.update_vo_status') as mock_update_vo_status:
```

**Pull request merged**